server/src/packages/rwcds-form/rwcds-form.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from '@nestjs/common';
+import { CrmService } from '../../crm/crm.service';
+import { PasFormService } from '../pas-form/pas-form.service';
+import { PACKAGE_ATTRS } from '../packages.controller';
+
+@Injectable()
+export class RwcdsFormService {
+  constructor(
+    private readonly crmService: CrmService,
+    private readonly pasFormService: PasFormService,
+  ) {}
+
+  async find(id) {
+    const { records: [rwcdsForm] } = await this.crmService.get(`dcp_rwcdsforms`, `
+      $filter=
+        dcp_rwcdsformid eq ${id}
+      &$expand=dcp_package($select=${PACKAGE_ATTRS.join(',')})
```

Comment:
@allthesignals Hmm, do we really need to expand packages on the rwcds form here? 

Wouldn't this be redundant, since the packages.service already requests the package?

I think expanding it here makes the final returned payload look like this:
```
  {
      ...package attrs
      ...RWCDS attrs,
      package: {
            ... package attrs
         },
        documents: {
           ....
        } 
     }
 ```

Reply:
Yeah, the package info is repeated in the nesting:
![image](https://user-images.githubusercontent.com/3311663/85342630-1acfd900-b4b9-11ea-936a-236d347f1c32.png)
Is this necessary?

Reply:
FWIW the package isn't included for serialization unless explicitly added. 

That said, probably not, no — I think this is a second request being added on after a package request. Makes sense. 

One other thing: we should add a `$select` here so we ask for explicit rwcds fields. That way, the request is fast.

Reply:
Per standup, this is an extra expand that is ignored by the JSON API Serializer anyway, so we can likely get rid of it. I'll look into it.